src/styles/utils/_variables.scss

```diff
@@ -4,6 +4,7 @@ $colors: (
   tertiary-grey: #f8f8f8,
   primary-white: #ffffff,
   pure-black: #000000,
+  text-light: #00000099,
```

Comment:
Why do we have this color? The length of `text-light` color in hex code is 8. Which means last two denote opacity of 99 (60%). If this is required case use more descriptive color RBGA color model.

Reply:
Removed